src/test/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionSimulatedData.java

```diff
@@ -22,6 +23,11 @@
  * @author David Benjamin
  */
 public final class AlleleFractionSimulatedData {
+    public static final ReadCountCollection TRIVIAL_TARGETS = new ReadCountCollection(
```

Comment:
I agree.  @LeeTL1220 if you approve I could allow `ReadCountCollection` to be empty pretty trivially.